adops/setting-up-prebid-with-the-appnexus-ad-server.md

```diff
@@ -103,6 +103,11 @@ Follow the creative setup instructions in [Add Creatives](https://console.appnex
 **Creative Expiration**  
 Note that creatives are automatically marked as inactive by the AppNexus systems after 45 days of inactivity.  This may happen to Prebid creatives since they are loaded relatively infrequently compared to other use cases.  For help with mitigating this issue, please contact your AppNexus representative.
 
+{: .alert.alert-warning :}
+**SafeFrame**
+If you want your creative to serve into a SafeFrame, this will need to be enabled on the site-side of the Prebid.js implementation rather than as a setting in the ad server.  There are details on this [page]({{site.github.url}}/dev-docs/examples/use-prebid-with-appnexus-ad-server.html) that the publisher's developers can reference to enabled this setting.  Additionally if the AppNexus ad server tags are configured to use SafeFrames, you **will** need to use the above creative template to properly render the creative.  Earlier versions of the Prebid.js creative template may not be fully SafeFrame compliant (if they are still in-use from older setups), so it is recommended to switch to the above template in this scenario.
```

Comment:
Change the sentence "There are details..." to "A developer can learn how to enable this setting for the publisher by referencing [Using Prebid.js with AppNexus Publisher Ad Server]({{site.github.url}}/dev-docs/examples/use-prebid-with-appnexus-ad-server.html)."

Reply:
Done